README.md

```diff
@@ -19,12 +19,12 @@ documentation].
 
 | Release | Release Date |   EOL    |
 |:-------:|:------------:|:--------:|
-|  v1.10  | Oct 18, 2022 | Jul 2023 |
 |  v1.11  | Jan 31, 2023 | Oct 2023 |
 |  v1.12  | Apr 25, 2023 | Jan 2024 |
-|  v1.13  | Late Jul '23 | Apr 2024 |
+|  v1.13  | Jul 27 2023  | Apr 2024 |
```

Comment:
dates look correct to me!! 

one tiny nit: can you include a `,` after the date number so it matches the other entries? e.g. `Jul 27, 2023` 😅